extensions/aproc/proc/ingest/drivers/impl/rapideye.py

```diff
@@ -131,23 +135,27 @@ def to_item(self, url: str, assets: list[Asset]) -> Item:
             ),
             assets=dict(map(lambda asset: (asset.name, asset), assets))
         )
+
+        if xml_path != self.xml_path:
+            os.remove(xml_path)
```

Comment:
The system should never delete any file. If it is really needed, then this must be documented and we must use the comment "!DELETE!".

https://docs.google.com/document/d/1UTm5JUZ8F9ptlcya9v88hzg-iPWxeuCCziAD_oycloo/edit?tab=t.0